derp/derphttp/derphttp_client.go

```diff
@@ -74,7 +74,7 @@ type Client struct {
 	privateKey key.NodePrivate
 	logf       logger.Logf
 	netMon     *netmon.Monitor // always non-nil
-	dialer     func(ctx context.Context, network, addr string) (net.Conn, error)
+	Dialer     func(ctx context.Context, network, addr string) (net.Conn, error)
```

Comment:
```suggestion
	dialer     func(ctx context.Context, network, addr string) (net.Conn, error)
```

I don't understand why this is exported when there is a `SetURLDialer` on Line 592?

Reply:
Because I missed `SetURLDialer`, I'll fix